packages/components/organisms/f-checkout/src/store/checkout.module.js

```diff
@@ -80,6 +80,8 @@ export default {
             const { data } = await axios.get(url, config);
 
             commit(UPDATE_STATE, data);
+
+            dispatch('analytics/updateAutofill', state, { root: true });
```

Comment:
I'm wondering if this should live inside the Vue component (push the analytics after we've called the action) rather than in the checkout store, just to keep things a bit more separated?

Reply:
Hey! So the aim was to keep as much out of the checkout component as possible so that it didn't get clogged up. More than happy to move it if you think it would be neater though.